Documentation/ApiOverview/SiteHandling/ErrorHandling/PageErrorHandler.rst

```diff
@@ -60,26 +59,26 @@ Examples
 Internal error page
 -------------------
 
-Show the internal page with uid 145 on errors with HTML status 404.
-The content of this page will be fetched via internal sub-request.
+Show the internal page with uid `145` on errors with HTML status code `404`.
 
-.. code-block:: yaml
+..  code-block:: yaml
 
-   errorHandling:
-     -
-       errorCode: 404
-       errorHandler: Page
-       errorContentSource: 't3://page?uid=145'
+    errorHandling:
+        -
+            errorCode: 404
+            errorHandler: Page
+            errorContentSource: 't3://page?uid=145'
 
 External error page
 -------------------
 
-Shows an external page on all errors with a HTML status not otherwise defined.
+Shows an external page on all errors with a HTTP status code not otherwise
+defined.
 
-.. code-block:: yaml
+..  code-block:: yaml
 
-   errorHandling:
-     -
-       errorCode: 0
-       errorHandler: Page
-       errorContentSource: 'https://typo3.org/404'
+    errorHandling:
+        -
+            errorCode: 0
+            errorHandler: Page
+            errorContentSource: 'https://typo3.org/404'
```

Comment:
Shall we use page-not-found instead of 404? In case there is a page with uid 404 but not a slug with 404 this forwards to page 404 which can cause strange effect.

Reply:
I thought about changing it to example.org. But I actually checked it, the page on https://typo3.org/404 exists and gives a 200 back ;-) So I thought is was by intention ...

Reply:
Yes, better to use example.org